index.html

```diff
@@ -157,6 +160,32 @@ <h1 class="title">About me</h1>
                             </div>
                         </div>
                     </section>
+                    <form action="https://formspree.io/f/moqzkbjd" method="post">
+
+                        <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+                        <label for="username"></label>
+                        <input type="text" id="username" name="username" placeholder="username" class="input" required maxlength="30">
+                        <br>
+                        <label for="email"></label>
+                        <input type="email" id="email" name="email" placeholder="email" class="input" required pattern=".*[.com]">
+                        <br><br>
+                        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="write me something..." required maxlength="500"></textarea>
+                        <br>
+                        <label for="getintouch"></label>
+                        <input type="submit"  value="Get in touch" id="getintouch">
```

Comment:
- [ ] Please kindly ensure that the button matches the given Interactions on Figma.

- Your design
![mobile](https://user-images.githubusercontent.com/41163571/229857403-392b3e18-5928-4903-8850-1a9d24fa97da.gif)

- Figma design
<img width="274" alt="button2" src="https://user-images.githubusercontent.com/41163571/227492511-ac99583d-add1-41a8-b99f-2b2f12771941.png">

- And it seems you are confusing clicked to disable 😃